### ISSUE TITLE
Looking for guidance regarding SC 1.4.4, Resize Text, on mobile native apps

### ISSUE THREAD

**Issue by mra11yx** [w3c/wcag2ict#4]
I've been trying to find guidance on SC 1.4.4 (Resize Text) and native apps on iOS and Android. 
Both Android OS and iOS support font scaling and display scaling to various degrees. iOS calls the display scaling feature "Display Zoom"; Android calls it "Display Size." (Of note -- I'm **not** referring to magnification functionality. Also, the display zoom functionality, from what I've seen, is not specifically advertised within the OS as being geared toward a specific group of users with disabilities. Both platforms allow access to it from the general Display settings pane.)
When testing against SC 1.4.4 for native apps, should the display size/zoom feature be taken into consideration? For example, if an app's text can only be resized up to 170% using the OS's font resize setting (let's say the app does not support the OS's largest font sizes, like "huge" and "enormous"), but 200% can be reached by also enabling the display zoom/size feature, would this be a failure?

Copying @mraccess77 and @haltersweb here.

**Comment by maryjom** [comment]
Moved to WCAG2ICT repository as the new TF will work to address issues tagged as WCAG2ICT from the WCAG repository.

**Comment by RadhikaTakyar** [comment]
Would like to follow this thread for any more updates on 1.4.4  for transactional Native Apps.

**Comment by maryjom** [comment]
Interpreting 1.4.4 Resize Text more broadly to non-web software and native mobile applications, it seems to be a reasonable approach that text be able to expand to 200% of the application's body text size. If a particular hardware platform and operating system do not support text resizing to be fully 200% the size, it also seems reasonable that the nearest supported increase of text size should be acceptable so that application authors are not required to fix the lack of support of increasing text size by the OS. IMO, the TF should explore the potential for a note (or two) to say this, as we've taken a similar approach in Note 7 in the [Applying 1.4.10 Reflow to Non-web Documents and Software](https://w3c.github.io/wcag2ict/#applying-sc-1-4-10-reflow-to-non-web-documents-and-software).

Something to note. WCAG2ICT is not at liberty to modify any of the normative aspects of WCAG success criteria's language. I keep seeing comments to the effect that WCAG2ICT will solve various problems of applying WCAG SC to non-web, but it cannot do so if it requires a normative change in WCAG to address the issue or make the requirement clearer. We can only change the WCAG language to replace web terminology with non-web terminology. We can also add notes to indicate thoughts on application that aligns with the WCAG intent for the SC.

**Comment by chaals** [comment]
I think @peterkorn's [approach](https://github.com/w3c/wcag2ict/issues/4#issuecomment-1148810146) is indeed the right one, but I would base it on "the smallest text size" rather than, or as well as, "body text".

"body text" is a pretty vague term in reality. It may make sense to include it alongside "smallest text used" to

1. ensure it is reasonably clear to people trying to do the right thing in normal circumstances, and
2. avoid enabling the legalistic approach of having 3 lines of small print half the size of body text that scale 200%, and then make body text scale only to 125%.

**Comment by mbgower** [comment]
> "body text" is a pretty vague term in reality.

Body text is an established domain description for the default level text, and seems to me the most appropriate term to use when discussing the text most essential for achieving the intent of Resize Text. A definition can always be generated, but I don't see why we'd abandon a typographic term established over centuries.

It is preferable to only considering the "smallest text used", which (where a smaller text exists) is going to be used for less important text than the default size on the page.

Body text is also the term used in a good chunk of the industry. It is endemic, for example, in [Apple's typography guidance](https://developer.apple.com/design/human-interface-guidelines/typography).

**Comment by bruce-usab** [comment]
@mitchellevan [offer to draft a response](https://lists.w3.org/Archives/Public/public-wcag2ict-tf/2024Jun/0017.html).

**Comment by maryjom** [comment]
The WCAG2ICT Task Force read through the long thread of comments and gleaned a few topics or points made in this issue which are listed below with our Task Force response.

**Point 1:** SC 1.4.4 Resize Text should require scaling in native apps to the extent supported by user settings of the platform.
**TF Answer to Point 1:** Where a platform does not support text enlargement up to 200%, non-web software content that has sufficiently large text would address the user need behind this success criterion. (Examples are given of what this minimum text size could be in [Section 508, clause 402.4](https://www.access-board.gov/ict/#402.4) or [ADA 2010 standard, clause 707.7.2](https://www.access-board.gov/ada/#ada-707_7_2).)

**Point 2:** We should allow most or all text to enlarge to less than 200%, especially when the text is initially not particularly small
**TF Answer to Point 2:** We've heard from the community, including members of the Low Vision Task Force, that in some contexts it can be better for users to enlarge already-large text to less than 200% of its starting size. For example, current guidance from both Google and Apple for app designers advises that as user settings double the size of body text in apps, heading text be increased to a size larger than body text but less than doubled. Such an approach would not meet SC 1.4.4, which is clear in its requirement that text can be resized to 200%. Policies outside of WCAG may allow meeting user needs in ways that do not meet the technical standards, e.g. through "equivalent facilitation".

**Point 3:** We've long held that PC magnifier programs are [assistive technology](https://www.w3.org/TR/WCAG/#dfn-assistive-technologies) and therefore not a method of meeting 1.4.4. Is the same true on other platforms?
**TF Answer to Point 3:** For Non-Web Documents and Software, features including [software provided by the platform ](https://www.w3.org/TR/wcag2ict-22/#dfn-assistive-technology)that provide a means of enlarging the text 200% (zoom or otherwise) without the loss of [content](https://w3c.github.io/wcag2ict/#content-on-and-off-the-web) or functionality, meet the intent of this success criteria.
 
Platform accessibility features, including platform software that, when applied, causes loss of content, including a reduction in the ability to distinguish characters, would not meet this success criteria.

**Comment by maryjom** [comment]
Closing as answered. See also the current guidance on [Applying SC 1.4.4 Resize Text to Non-web Documents and Software](https://w3c.github.io/wcag2ict/#applying-sc-1-4-4-resize-text-to-non-web-documents-and-software).
